docs/user-guide/permissions/github.md

```diff
@@ -0,0 +1,12 @@
+Atomist automates activities like project creation and code changes that operate on code in GitHub repos. Atomist also makes it easy to get relevant, actionable notification of GitHub activity like new commits or issues. In order to do that, the Atomist GitHub integration requests the following permissions:
```

Comment:
"In order to do that, the Atomist GitHub integration requests the following permissions:"

I'd replace this with:

"To enable this, the Atomist GitHub integration requests the following permissions:"